arithmetization/src/main/java/net/consensys/linea/zktracer/module/mmu/instructions/AnyToRamWithPadding.java

```diff
@@ -230,10 +230,7 @@ private void sharedRow2(final HubToMmuValues hubToMmuValues) {
     if (mixed) {
       transferSize =
           (hubToMmuValues.referenceSize() - hubToMmuValues.sourceOffsetLo().longValueExact());
-      paddingSize =
-          (hubToMmuValues.size()
-              - (hubToMmuValues.referenceSize()
-                  - hubToMmuValues.sourceOffsetLo().longValueExact()));
+      paddingSize = (hubToMmuValues.size() - transferSize);
```

Comment:
Was that a bug ?

Reply:
no, just a simplification as `transferSize = (hubToMmuValues.referenceSize()
                  - hubToMmuValues.sourceOffsetLo().longValueExact())`